src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-events-integration.mdx

```diff
@@ -79,9 +79,9 @@ Then, to view the pod details:
 
 ## Optional: Collect metrics of the event collector [#events-mentrics]
 
-To collect metric data for the event collector itself, install the New Relic Prometheus OpenMetrics integration. To get started:
+To collect metric data for the event collector itself, install the New Relic Prometheus Agent integration. To get started:
```

Comment:
@nbaenam "Prometheus agent" in this line and the next.

Reply:
Done! Good catch, @rhetoric101!